modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/analytics/config/identity_event_analytics_enabled.toml

```diff
@@ -17,15 +16,27 @@ connection_password = "admin"
 base_dn = "dc=wso2,dc=org"      #refers the base dn on which the user and group search bases will be generated
 
 [database.identity_db]
-#type = "h2"         # Infers the local H2 DB connection details for 'DEV' pattern. SHOULD BE CHANGED TO PRODUCTION READY DB FOR PRODUCTION
+driver = "org.h2.Driver"
+url = "jdbc:h2:./repository/database/WSO2IDENTITY_DB;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=60000"
+validation_query = "SELECT 1"
+username = "wso2carbon"
+password = "wso2carbon"
 
 [database.shared_db]
-#type = "h2"         # Infers the local DB connection details. SHOULD BE CHANGED TO PRODUCTION READY DB FOR PRODUCTION
+driver = "org.h2.Driver"
+url = "jdbc:h2:./repository/database/WSO2SHARED_DB;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=60000"
+validation_query = "SELECT 1"
+username = "wso2carbon"
+password = "wso2carbon"
 
 [keystore.tls]
 name = "wso2carbon.jks"
 password = "wso2carbon"
 
+[truststore]
+name = "client-truststore.jks"
```

Comment:
shall we add this to default file?

Reply:
It is added in the default file. But in production users should change this and shouldn't this config visible to the uesrs?

Reply:
Fixed